squarelet/organizations/tests/test_wix.py

```diff
@@ -137,3 +141,203 @@ def test_sync_wix(self, requests_mock, user):
 
         for url, response in zip(urls, requests_mock.request_history):
             assert response.url == url
+
+    @pytest.mark.django_db()
+    def test_create_contact(self, mocker, user):
+        """Test the create_contact function using Contacts API v4"""
+        contact_id = str(uuid.uuid4())
+        mock_response = mocker.Mock()
+        mock_response.json.return_value = {"contact": {"id": contact_id}}
+        mock_response.raise_for_status.return_value = None
+        mock_response.status_code = 200
+        mock_post = mocker.patch("requests.post", return_value=mock_response)
```

Comment:
The `requests-mock` library provides an easier way to mock requests:
https://requests-mock.readthedocs.io/en/latest/overview.html
https://github.com/MuckRock/squarelet/blob/master/squarelet/organizations/tests/test_wix.py#L26